src/components/WorkoutCard/WorkoutCard.spec.js

```diff
@@ -33,12 +35,20 @@ const props = {
 }
 
 describe('WorkoutCard', () => {
-  it('renders a workout´s title, description and duration', () => {
+  it('renders a workout´s title, description, duration and show/hide button', () => {
     render(<WorkoutCard {...props} />)
     expect(screen.getByText(/On FTP-Light/i)).toBeInTheDocument()
     expect(
       screen.getByText(/Ride right on FTP for a period of time/i)
     ).toBeVisible()
     expect(screen.getByText(/40/)).toBeInTheDocument()
+    expect(screen.getByRole('button')).toBeInTheDocument()
+  })
+
+  it('has a button that calls onClick prop when clicked', () => {
```

Comment:
I think you want to test whether the section is visible after a button click or not. For that you will need to render your WorkoutCard, find the Button in there, click it and then check whether the section is visible or not. You don't need a callback function then. If you have a questions about it, please ask :)